src/service calls/sevices.jsx

```diff
@@ -0,0 +1,67 @@
+import axios from 'axios';
+import React from 'react';
+
+
+const addToCart = async (product,{encodedToken,authDispatch,navigator })=>{
+    try {
+        const response = await axios.post("/api/user/cart",{product},{headers:{ authorization : encodedToken }})
```

Comment:
you can destructure `data` from response `const  {data} = await ...`

Reply:
i always forgot this , thanks for pointing ou again